### DISCUSSION TITLE
Vaultwarden LXC - Update/Upgrade script

### DISCUSSION THREAD

**Discussion by vdias** [tteck/Proxmox#1082] [General]
Wanted to upgrade my vaultwarden installation, and as usual i always go for the update script for a quick deploy... but missing it on this LXC... any special trick for this one?

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/issues/914

**Comment by vdias** [comment]
Sorry for that detail... nice work on v5.

But now i'm facing a error...
 
_Stopped Vaultwarden
Updating VaultWarden to 1.27.0 (Patience)...
[ERROR] in line 347: exit code 101: while executing command cargo build --features "sqlite,mysql,postgresql" --release &> /dev/null_


CT Details:
CPU - 2vCores
RAM - 3GB
DISK - 5.50gb (used) of 5.82gb (total)

Seems that disk is getting full after executing update script twice... how can i cleanup to realunch it again?

**Reply by tteck** [reply]
To build the new version, you will require additional storage space. However, once the binary is built and the old version is replaced, the storage space required will decrease. Here's a look at my current storage: 

![Screenshot 2023-02-20 6 25 26 AM](https://user-images.githubusercontent.com/21342670/220094491-6094b596-e612-4f52-893c-cd111d7da50f.png)

Just a quick note: when building the new binary, I'm using 6 virtual CPUs and 4 GB of RAM. This helps to ensure that the build process runs smoothly and efficiently.

**Reply by waleedHubaishi** [reply]
If anyone else facing the same error you mentioned and the resources suggested are correctly set, try as a 'root'
